PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c

```diff
@@ -660,6 +664,8 @@ PcRtcSetTime (
   RegisterB.Bits.Set = 0;
   RtcWrite (RTC_ADDRESS_REGISTER_B, RegisterB.Data);
 
+  RegisterA.Bits.Dv = 0x2;

```

Comment:
This code needs a comment block to explain what a Dv value of 0x02 means and why it is required to have a value of 0x02 when RTC is running normally.  Since the state of .Dv is never read, how does this code know that setting to 0x02 matches the original value before temporarily setting to 0x07?  Should the logic above read and save the .Dv state and restore it here instead of using hard coded 0x02 value?